docs/en/connector-v2/sink/ObsFile.md

```diff
@@ -0,0 +1,260 @@
+# ObsFile
```

Comment:
After discussion in the community, we have optimized the format requirements of the document, which you can refer to：`https://github.com/apache/incubator-seatunnel/issues/4544`

Reply:
Ok